### DISCUSSION TITLE
Selfhosted - limited aliases

### DISCUSSION THREAD

**Discussion by lgwapnitsky** [simple-login/app#838] [Q&A]
I would like to test this surface against a competitor service comma but the competitor allows for unlimited aliases on a self hosted instance. Is there a reason for only 15 aliases when self-hosting the community version?

**Comment by nguyenkims** [comment]
You can have unlimited aliases on SimpleLogin by following this step https://github.com/simple-login/app/blob/99ff4c6f88198423d89e53ea610936508c6da35e/README.md#enjoy
